swap_meet/vendor.py

```diff
@@ -1,2 +1,83 @@
+from .item import Item
+
+
 class Vendor:
-    pass
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
We can get rid of the else clause by flipping the logic and putting the 'true' logic more into focus.

```suggestion
    def remove(self, item):
        if item not in self.inventory:
           return False
        self.inventory.remove(item)
        return item
```